Assets/MixedRealityToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -23,6 +23,17 @@ namespace MixedRealityToolkit.SpatialMapping
     [RequireComponent(typeof(Interpolator))]
     public class TapToPlace : MonoBehaviour, IInputClickHandler
     {
+        /// <summary>
+        /// The collider reference point used to place the object.
+        /// </summary>
+        public enum ColliderPlacement
```

Comment:
Per code style guidelines (see #1846), please move public enums outside of the class. I also recommend naming it TapToPlaceColliderPlacement to make it clear which class it is for (unless you believe it has a more generic usage.